ArcGISRuntimeSDKQt_CppSamples/Layers/ListKmlContents/ListKmlContents.qml

```diff
@@ -33,7 +33,10 @@ Item {
             height: childrenRect.height
             color: "lightgrey"
 
-            ColumnLayout {
+            GridLayout {
+                columns: 1
```

Comment:
Columns should be 2.

Remove the `RowLayout`s. `myListView` should just span 2 columns using the injected property `Layout.columnSpan`.


Remove all instances of manually set width and height. Use `Layout.fillWidth` on the textRectangle.